src/Behaviors/Behaviors/Shared/Entry/EntryEmptyValidation.cs

```diff
@@ -74,24 +78,25 @@ protected override void OnDetachingFrom(Entry bindable)
             base.OnDetachingFrom(bindable);
         }
 
-        void HandleTextChanged(object sender, TextChangedEventArgs e)
+        private void HandleTextChanged(object sender, TextChangedEventArgs e)
         {
             var text = e?.NewTextValue ?? string.Empty;
             IsValid = !string.IsNullOrWhiteSpace(text);
 
             var entry = sender as Entry;
 
             if (entry == null)
+            {
```

Comment:
braces

Reply:
A rule specify you have to put braces. :)
You can disable it if you want. It's a guideline.